assets/src/checkout/pages/BamboraPayPage.tsx

```diff
@@ -25,6 +26,7 @@ import CardForm, {
   classNames as cardFormClassNames
 } from "../components/BamboraCardForm";
 import Summary from "../components/Summary";
+import { CardTypeBrand } from "../../auth/__generated__/graphql";
```

Comment:
probably import from a non `auth` generated file just because it's not related to auth. i know we have these generated files duplicated in here so all good

Reply:
done

Reply:
@acodegirl did you push this change yet? it looks like it's still importing from `../../auth`?